<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimControllerPrefWidget</class>
 <widget class="QWidget" name="SimControllerPrefWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulation Controller</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="statusBarGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Status Bar</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="mShowSimModeCheckBox">
          <property name="text">
           <string>Simulation Mode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mShowSimStateCheckBox">
          <property name="text">
           <string>Simulation State</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mShowClockRateCheckBox">
          <property name="text">
           <string>Clock Rate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mShowWallTimeCheckBox">
          <property name="text">
           <string>Wall Time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mShowSimTimeCheckBox">
          <property name="text">
           <string>Simulation Time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mShowProgressBarCheckBox">
          <property name="text">
           <string>Progress Bar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mShowTimeBehindCheckBox">
          <property name="text">
           <string>Time Behind</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Wall Time Specification</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QRadioButton" name="mLocalTimeSpecButton">
             <property name="text">
              <string>Local Time</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="mUTC_TimeSpecButon">
             <property name="text">
              <string>UTC (Coordinated Universal Time)</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QRadioButton" name="mUTC_OffsetTimeSpecButton">
               <property name="text">
                <string>UTC Offset</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="mUTC_OffsetHrsBox">
               <property name="suffix">
                <string> hrs</string>
               </property>
               <property name="minimum">
                <number>-14</number>
               </property>
               <property name="maximum">
                <number>14</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="mUTC_OffsetMinsBox">
               <property name="suffix">
                <string> mins</string>
               </property>
               <property name="maximum">
                <number>59</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="networkGroupBox">
     <property name="title">
      <string>Networking</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QCheckBox" name="mSendDIS_PDUsCheckBox">
        <property name="text">
         <string>Send Stop/Start DIS PDUs on Pause/Resume commands</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="mAutoPauseGroupBox">
        <property name="title">
         <string>Automatically Pause when remote simulations fall behind</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="mFallingBehindLabel">
           <property name="text">
            <string>Threshold:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="mFallingBehindSpinBox">
           <property name="suffix">
            <string> seconds</string>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="toolbarGroupBox">
     <property name="title">
      <string>Toolbar</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_3">
          <property name="toolTip">
           <string>Move action up</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
          <property name="arrowType">
           <enum>Qt::UpArrow</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton">
          <property name="toolTip">
           <string>Remove action from toolbar</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
          <property name="arrowType">
           <enum>Qt::LeftArrow</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_2">
          <property name="toolTip">
           <string>Add action to toolbar</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
          <property name="arrowType">
           <enum>Qt::RightArrow</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_4">
          <property name="toolTip">
           <string>Move action down</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
          <property name="arrowType">
           <enum>Qt::DownArrow</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="4">
       <widget class="QListWidget" name="mCurrentActionsListWidget"/>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="mCurrentActionsLabel">
        <property name="text">
         <string>Current Actions</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QListWidget" name="mAvailableActionsListWidget"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="mAvailableActionsLabel">
        <property name="text">
         <string>Available Actions</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mShowSimModeCheckBox</tabstop>
  <tabstop>mShowSimStateCheckBox</tabstop>
  <tabstop>mShowClockRateCheckBox</tabstop>
  <tabstop>mShowWallTimeCheckBox</tabstop>
  <tabstop>mShowSimTimeCheckBox</tabstop>
  <tabstop>mShowProgressBarCheckBox</tabstop>
  <tabstop>mShowTimeBehindCheckBox</tabstop>
  <tabstop>mLocalTimeSpecButton</tabstop>
  <tabstop>mUTC_TimeSpecButon</tabstop>
  <tabstop>mUTC_OffsetTimeSpecButton</tabstop>
  <tabstop>mUTC_OffsetHrsBox</tabstop>
  <tabstop>mUTC_OffsetMinsBox</tabstop>
  <tabstop>mSendDIS_PDUsCheckBox</tabstop>
  <tabstop>mAvailableActionsListWidget</tabstop>
  <tabstop>mCurrentActionsListWidget</tabstop>
  <tabstop>toolButton_3</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>toolButton_2</tabstop>
  <tabstop>toolButton_4</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
